### PR TITLE
skip over false tags

### PR DESCRIPTION
**Pull request by vladak**
With this change the tag rebuild for the linux-mainline repository finishes fine, with some 687 `tagList` entries currently.

### CODE REVIEW

File: opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java [hunk 1]
```diff
@@ -649,7 +654,7 @@ private void rebuildTagList(File directory) {
             // In case of partial success, do not null-out tagList here.
         }
 
-        if (LOGGER.isLoggable(Level.FINEST)) {
+        if (LOGGER.isLoggable(Level.FINER)) {
```
Comment (ahornace) [inline opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java:657]: Why this change? It does not make much sense to check for higher log level, right?

Comment (vladak) [reply opengrok-indexer/src/main/java/org/opengrok/indexer/history/GitRepository.java:657]: I think it should be higher than the `FINEST` log above.
